### PR TITLE
Issue #SB-16745 fix: Resource: Privacy option

### PR DESCRIPTION
**Pull request by rahulshukla**

### CODE REVIEW

**Review by pallakartheekreddy — DISMISSED** [review]

**Review by pallakartheekreddy — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "body-parser": "*",
     "codacy-coverage": "^3.4.0",
     "express": "^4.14.0",
+    "global": "^4.4.0",
```
Comment (rahulshukla) [inline package.json:19]: @venkateshwarans @pallakartheekreddy  We have added this NPM as a dependency, please look into this

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
+sonar.projectName=sunbird-collection-editor
```
Comment (rahulshukla) [inline sonar-project.properties:1]: This is for the Sonarqube check. This won't break anything in the collection editor. it is just for Sonarcube continuous inspection which will analyze project code on each pull request.

File: webpack.collection.config.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 
 var corePlugins = [
     "org.ekstep.conceptselector-1.1",
-    "org.ekstep.assetbrowser-1.2",
+    "org.ekstep.assetbrowser-1.3",
```
Comment (rahulshukla) [inline webpack.collection.config.js:15]: This is for asset browser bump up, which has license support

File: webpack.collection.config.js [hunk 4]
```diff
@@ -143,6 +143,9 @@ module.exports = {
             'jquery': path.resolve('./node_modules/jquery/dist/jquery.js'),
             'angular': path.resolve('./app/bower_components/angular/angular.js'),
             'iziToast': path.resolve('./app/bower_components/izitoast/dist/js/iziToast.min.js'),
+            'video.js': path.resolve('./plugins/org.ekstep.video-1.5/editor/libs/video.js'),
```
Comment (rahulshukla) [inline webpack.collection.config.js:146]: @venkateshwarans @pallakartheekreddy  please have a look into this.

**Pull request merged**